### PR TITLE
Added Plone newsletter sign up in footer #182

### PR DESCRIPTION
**Pull request by Adityaadpandey**
Added a Plone newsletter sign up page link to Plone.org in the footer #182

### CODE REVIEW

File: frontend/src/customizations/volto/components/theme/Footer/Footer.jsx
Comment (stevepiercy) [inline frontend/src/customizations/volto/components/theme/Footer/Footer.jsx]: The link is relative to the site root.

```suggestion
                    <a href="/news-and-events/join-the-plone-newsletter">
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by stevepiercy** [comment]
@Adityaadpandey please read and follow [Contributing](https://6.docs.plone.org/contributing/index.html). We cannot merge this PR without your signed Plone Contributor Agreement. Thank you!

**Comment by Adityaadpandey** [comment]
Thank you for the help, as now i read and signed the contract.

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
@Adityaadpandey we need to wait for your Plone Contributor Agreement to get approved, then we can merge. Thank you for your contribution!

**Comment by Adityaadpandey** [comment]
The request for contribution on plane has been approved

**Comment by stevepiercy** [comment]
@Adityaadpandey it appears that you did not complete the process by accepting the invitation.

**Comment by Adityaadpandey** [comment]
Sorry for the inconvenience now i have done that is there anything else left to do

**Comment by stevepiercy** [comment]
Needs review by @plone/marketing-communications team.

**Comment by tkimnguyen** [comment]
This is a nice idea. Thank you @Adityaadpandey

File: frontend/src/customizations/volto/components/theme/Footer/Footer.jsx
Comment (davisagli) [inline frontend/src/customizations/volto/components/theme/Footer/Footer.jsx]: @Adityaadpandey Please update the link to use the Link component like the other internal links in this menu. This way the navigation happens via the Volto router and doesn't need a full page reload.

**Review by davisagli — CHANGES_REQUESTED** [review]

File: frontend/src/customizations/volto/components/theme/Footer/Footer.jsx
Comment (davisagli) [inline frontend/src/customizations/volto/components/theme/Footer/Footer.jsx]: @Adityaadpandey Look carefully -- the `Link` component uses `to` instead of `href`

**Review by Adityaadpandey — COMMENTED** [review]
made the changes as asked for the use  Link component so doesn't need a full page reload.

**Review by davisagli — APPROVED** [review]

**Review by astridjuizi — COMMENTED** [review]
Looks good and in the correct position.

**Comment by fredvd** [comment]
LGTM, @astridjuizi also checked the testing site.  I'll create a release this evening to update the live site.  I see there are also some other changes on main that might not have landed in the changelog, these will also go live, I'll try to update the changelog.

**Comment by fredvd** [comment]
Pushed to live plone.org in release 1.0.17.

**Pull request merged**
